options/rtdl/generic/linker.cpp

```diff
@@ -683,8 +683,10 @@ void ObjectRepository::_parseDynamic(SharedObject *object) {
 		case DT_TLSDESC_PLT: case DT_TLSDESC_GOT:
 			break;
 		default:
```

Comment:
If you could please add a comment here as well explaining why we have this if statement here, that would be great. I just know someone is going to take a look at this code in the future and be extremely confused, and I'd like to avoid that.